ros/angel_system_nodes/angel_system_nodes/annotation_event_monitor.py

```diff
@@ -0,0 +1,112 @@
+import threading
+from threading import Lock
+
+from pynput import keyboard
+import rclpy
+from rclpy.node import Node
+
+from angel_msgs.msg import AnnotationEvent
+
+
+class AnnotationEventMonitor(Node):
+    """
+    ROS node that monitors the keyboard to generate AnnotationEvent
+    messages. Event messages for annotation start and stop are generated
+    when the right arrow key is pressed. Event messages for error start
+    and stop are are generated when the left arrow key is pressed.
```

Comment:
I know we're not also running the task monitor node to technically conflict, but there is a little mental overlap being created by using the same buttons. I'm entering a little into bikesheding territory here, but I might recommend monitoring some different keys for this.

Reply:
Ok, good idea. I switched it to use the up/down arrow keys instead.

Reply:
Thanks for the change. In an idea world, we should have a document overviewing the UI/UX of various components, which would include what keys are recognized by what nodes, so as to be a reference for users as well as development to inform what keys are "reserved" or not. I don't think this is a priority, but something to keep in the back of your mind. This would also somewhat coincide with breaking the README into more modular documentation that hasn't happened yet.